### ISSUE TITLE
Creating instances of C++ templates with parameter packs

### ISSUE THREAD

**Issue by iexavl** [bytedeco/javacpp#773]
I tried to make a struct with a parameter pack template as such:
```
template<typename... Args>
struct Foo
{
    virtual void bar(Args... args) {}
};
```
and I made a function that has specializes this struct:

`void fooFunc(Foo<int, double> b) {...}`

I put it in an InfoMapper:

`infoMap.put(new Info("Foo<int,double>").pointerTypes("TestType").virtualize());`

and it did generate a function that takes the correct type 

`public native void pPackTest(@ByVal TestType b);`

But failed to generate the actual .java file (TestType.java)
Is there something I can do about this or is this just a limitation of the parser?

**Comment by saudet** [comment]
You'll need to set Info.define() for it to do that.

**Comment by saudet** [comment]
Right, that's probably not going to work with template parameter packs... You can still define wrapper classes manually though.
